src/rvpt/rvpt.cpp

```diff
@@ -6,9 +6,12 @@
 #include <fstream>
 
 #include <glm/glm.hpp>
+#include <glm/glm/ext.hpp>
 #include <nlohmann/json.hpp>
 
-RVPT::RVPT(Window& window) : window_ref(window)
+#include <random>
```

Comment:
Prefer to put standard library headers above external libraries.
My order is standard C++ libraries, standard C libraries, external libraries, header files not in the local folder, local header files. (basically, the order of 'C++, C, external libraries, internal libraries but not in the same 'module', then in-'module' headers.